doc/onboarding-extras.md

```diff
@@ -78,6 +78,29 @@ Please use these when possible / appropriate
   git checkout $(git show -s --pretty='%T' $(git show-ref -d $(git describe --abbrev=0) | tail -n1 | awk '{print $1}')) -- test; make -j4 test
   ```
 
+### LTS/Version labels
+
+We use labels to keep track of which branches a commit should land on:
+
+* `dont-land-on-v?.x`
+  * For changes that do not apply to a certain release line
+  * Also used when the work of backporting a change outweighs the benefits
+* `land-on-v?.x`
+  * Used by releasers to mark a PR as scheduled for inclusion in an LTS release
+  * Applied to the original PR for clean cherry-picks, to the backport PR otherwise
+* `backport-requested-v?.x`
+  * Used to indicate that a PR needs a manual backport to a branch in order to land on it
+  * Will be replaced by either `dont-land-on-v?.x` or `backported-to-v?.x`
+* `backported-to-v?.x`
+  * Applied to PRs for which a backport PR has been made
+* `lts-watch-v?.x`
+  * Applied to PRs which the LTS working group should consider including in a LTS release
+  * Does not indicate that any specific action, but can be effective as messaging to non-collaborators
```

Comment:
Is there something missing here? Like 'Does not indicate that any specific action is needed...'.